airflow/configuration.py

```diff
@@ -1531,14 +1531,28 @@ def initialize_config() -> AirflowConfigParser:
         if local_conf.getboolean("core", "unit_test_mode"):
             local_conf.load_test_config()
 
-    # Make it no longer a proxy variable, just set it to an actual string
+    old_webserver_config = AIRFLOW_HOME + "/webserver_config.py"
+    # Prioritise airflow webserver config that is present in the config location
+    webserver_config_in_config = AIRFLOW_HOME + "/config/webserver_config.py"
```

Comment:
Please use os.path or pathlib instead.

Reply:
@uranusjr where should i use either of those modules? I am defining some constants here and then checking if those paths exist using `os.path.isfile`

Reply:
Ok sure. Used `os.path.join` here